### ISSUE TITLE
Cannot install sr-speedbar

### ISSUE THREAD

**Issue by jeanbaptistelab** [dimitri/el-get#2152]
Using what I think is a no-frills, based on the examples configuration, I can't for the life of me get the sr-speedbar package to install, I'm getting the following error upon start up:

```
Warning (initialization): An error occurred while loading `/home/jbl/.emacs.d/init.el':
Wrong number of arguments: <=, 3

which yields the following stacktrace:
Debugger entered--Lisp error: (wrong-number-of-arguments <= 3)
  <=(200 200 299)
  el-get-http-install(sr-speedbar "https://raw.github.com/emacsmirror/emacswiki.org/master/sr-speedbar.el" el-get-post-install)
  el-get-emacswiki-install(sr-speedbar nil el-get-post-install)
  el-get-do-install(sr-speedbar)
  el-get-init-and-install((ibuffer-vc vala-mode rainbow-mode json-mode json-reformat scss-mode sr-speedbar))
  el-get(sync (ibuffer-vc vala-mode rainbow-mode json-mode json-reformat scss-mode sr-speedbar))
  eval-buffer(#<buffer  *load*> nil "/home/jbl/.emacs.d/init.el" nil t)  ; Reading at buffer position 2959
  load-with-code-conversion("/home/jbl/.emacs.d/init.el" "/home/jbl/.emacs.d/init.el" t t)
  load("/home/jbl/.emacs.d/init" t t)
  #[0 "\205\262�	\306=\203�\307\310Q\202;
[TRUNCATED]
 command-line()
  normal-top-level()
```

Any clues as to the cause of this error and how to fix it?

**Comment by npostavs** [comment]
oops, I didn't notice that `<=` accepting multiple arguments is new in Emacs 24.4.

**Comment by npostavs** [comment]
Should be fixed in f4e71160b8602667a5ea502953a367b85ccf7689
